package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@simpleviewinc/appetite",
+  "version": "0.0.1",
+  "description": "appetize.io api wrapper",
+  "main": "index.js",
+  "author": "mpcarolin.dev@gmail.com",
+  "repository": "https://github.com/simpleviewinc/appetite",
+  "license": "MIT",
+  "scripts": {
+    "demo": "node index.js",
+    "test": "yarn jest --config ./configs/jest.config.js",
+    "apt:test": "node ./scripts/apt-test.js"
+  },
+  "dependencies": {
+    "@keg-hub/jsutils": "8.6.0",
+    "axios": "^0.21.1",
+    "form-data": "^4.0.0",
+    "rimraf": "3.0.2",
+    "tar-fs": "2.1.1"
```

Comment:
Not blocking but I'm pretty sure this is not used, I could be wrong though

Reply:
updated